_src/_scss/partials/_tufte.scss

```diff
@@ -0,0 +1,79 @@
+input.margin-toggle { 
+    display: none; 
+}
+
+label.sidenote-number { 
+    display: inline; 
+}
+
+label.margin-toggle:not(.sidenote-number) { 
+    display: none; 
+}
+
+//Sidenote styling
+.sidenote, 
+.marginnote { 
+    float: right;
+    clear: right;
+    margin-right: -60%;
+    width: 50%;
+    margin-top: 0;
+    margin-bottom: 0;
+    font-family: $serif-font-family;
+    font-size: 0.88rem;
+    opacity: 0.75;
+    vertical-align: baseline;
+    position: relative;
+}
+
+.sidenote-number { 
+	counter-increment: sidenote-counter; 
+}
+
+.sidenote-number:after, 
+.sidenote:before { 
+	content: counter(sidenote-counter) " ";
```

Comment:
Seems to be an embedded tab here and at several points in the document (copy-paste error?). The existing format should prevail: *two* (not four) spaces per indent, and no tabs.